VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -60,6 +62,14 @@ public UserAPI(@NonNull RestAdapter restAdapter) {
     public Account getAccount(@NonNull String username) throws RetroHttpException {
         final Account account = userService.getAccount(username);
         EventBus.getDefault().post(new AccountDataLoadedEvent(account));
+
+        // Store the logged in user's ProfileImage
+        PrefManager prefManager = new PrefManager(MainApplication.instance(), PrefManager.Pref.LOGIN);
+        if (prefManager.getCurrentUserProfile() != null) {
+            if (username.equals(prefManager.getCurrentUserProfile().username)) {
+                prefManager.put(PrefManager.Key.PROFILE_IMAGE, gson.toJson(account.getProfileImage()));
+            }
+        }
```

Comment:
In order to keep the persisted profile image fields up to date, you'll need to also update it from the `setProfileImage()`, and `deleteProfileImage()` methods, and maybe even from the `updateAccount()` method as well.